src/openrct2/peep/Peep.cpp

```diff
@@ -5800,7 +5806,6 @@ static void peep_update_1(rct_peep * peep)
     peep->destination_x         = peep->x;
     peep->destination_y         = peep->y;
     peep->destination_tolerance = 10;
-    peep->var_76                = 0;
```

Comment:
Why this change?

Reply:
It's not used at any point. I did quite a search on this before removing it